waiter/src/waiter/util/utils.clj

```diff
@@ -133,18 +133,13 @@
 (defn stringify-elements
   [k v]
   (cond
-    (vector? v) (mapv (partial stringify-elements k) v)
-    (keyword? v) (keyword->str v)
+    (contains? #{true false nil} v) v
+    (map? v) v
+    (number? v) v
+    (sequential? v) (mapv (partial stringify-elements k) v)
     (instance? DateTime v) (du/date-to-str v)
-    (instance? UUID v) (str v)
-    (instance? Pattern v) (str v)
-    (instance? PersistentQueue v) (vec v)
-    (instance? ManyToManyChannel v) (str v)
-    (instance? Process v) (str v)
-    (instance? ValidationError v) (str v)
-    (= k :time) (str v)
```

Comment:
We can drop this `(= k :time)` because it's already covered by our `DateTime` conversion.